packages/sharing/vite.config.ts

```diff
@@ -71,6 +71,9 @@ export default defineConfig(({ mode }) => ({
   },
   define: {
     "process.env.NODE_ENV": JSON.stringify(mode),
+    "process.env.STREAMLIT_VERSION": JSON.stringify(
```

Comment:
What provides the `STREAMLIT_VERSION` environment variable?
Looks like it's now working just because of the fallback value. Delete the fallback value for the "fail early" principle.

Also, the defined const name should be just `STREAMLIT_VERSION` without `process.env.`. It should be treated just as a const during the build.